addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -45,24 +51,47 @@ def unlink(self):
                 # Note: invitation depends on field `rtc_inviting_session_id` so the cancel must be
                 # done before the delete to be able to know who was invited.
                 channel._rtc_cancel_invitations()
+                # If there is no member left in the RTC call, we remove the SFU channel uuid as the SFU
+                # server will timeout the channel. It is better to obtain a new channel from the SFU server
+                # than to attempt recycling a possibly stale channel uuid.
+                channel.sfu_channel_uuid = False
         notifications = [(channel, 'discuss.channel/rtc_sessions_update', {
             'id': channel.id,
             'rtcSessions': [('DELETE', [{'id': session_data['id']} for session_data in sessions_data])],
         }) for channel, sessions_data in self._mail_rtc_session_format_by_channel().items()]
+        session_ids_by_channel = defaultdict(list)
         for rtc_session in self:
+            sfu_channel_uuid = rtc_session.channel_id.sfu_channel_uuid
+            if not sfu_channel_uuid:
+                continue
+            session_ids_by_channel[sfu_channel_uuid].append(rtc_session.id)
             target = rtc_session.guest_id or rtc_session.partner_id
             notifications.append((target, 'discuss.channel.rtc.session/ended', {'sessionId': rtc_session.id}))
         self.env['bus.bus']._sendmany(notifications)
-        return super().unlink()
+        return_value = super().unlink()
+        if self.env.context.get('notify_rtc_server'):
+            url = channels._get_rtc_server_url()
+            if url:
+                key = jwt.get_rtc_key(self.env)
+                try:
+                    requests.post(
+                        url + '/disconnect',
+                        json={"jsonWebToken": jwt.sign({'sessionIdsByChannel': session_ids_by_channel}, key=key, ttl=20, algorithm=jwt.ALGORITHM.HS256)},
```

Comment:
Is it a standard practice to set the `jsonWebToken` in the payload for a `POST` request and in the `headers` for a `GET` request?
Just asking because there are only 2 controllers called from the Python side and the signature is handled differently. I would have thought the headers method would be appropriate for `GET` and `POST`.

Reply:
Ah I don't know if there is a standard practice in that case, It felt like in my GET route, it was just to authenticate, whereas in the POST it contains data that I do process